mcp_zammad/server.py

```
@@ -54,6 +55,27 @@
 )
 
 
+class AttachmentDeletionError(Exception):
+    """Raised when attachment deletion fails."""
+
+    def __init__(self, ticket_id: int, article_id: int, attachment_id: int, reason: str):
```

Ruff (0.14.4)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)